### ISSUE TITLE
update-container-image fails to update multiple images

### ISSUE THREAD

**Issue by parkerjm** [CircleCI-Public/aws-eks-orb#18]
This may be an issue with the kubernetes orb, but in the latest version of the eks orb (0.2.2), supplying a list of multiple images delimited by spaces fails to update multiple images. Instead, it attempts to set the first container's image to the entire string that comes after the first `=` character. 

For instance, here's my circleci config.yml snippet:
```      - aws-eks/update-container-image:
          name: Staging Deploy
          requires:
            - Deploy to staging
          aws-region: us-east-2
          cluster-name: staging
          resource-name: deployment.v1.apps/my-app-staging
          container-image-updates: my-app=my-org/my-app:$CIRCLE_SHA1 my-app-job-runner=my-org/my-app:$CIRCLE_SHA1
          record: true
```

and here is the outcome:
```
Containers:
  my-app:
    Container ID:
    Image:          my-org/my-app:f3c65ebb0602966922eeceb92af37cdf6222d375 my-app-job-runner=my-org/my-app:f3c65ebb060296692
2eeceb92af37cdf6222d375
```

which of course leads to kubernetes image pull failures:
```
Warning  InspectFailed   10s (x5 over 26s)  kubelet, ip-192-168-9-21.us-east-2.compute.internal  Failed to apply default image tag "my-org/my-app:f3c65ebb0602966922eeceb92af37cdf6222d375 my-app-job-runner=my-org/my-app:f3c65ebb0602966922eeceb92af37cdf6222d375": couldn't parse image reference "my-org/my-app:f3c65ebb0602966922eeceb92af37cdf6222d375 my-app-job-runner=my-org/my-app:f3c65ebb0602966922eeceb92af37cdf6222d375": invalid reference format
```

I've tried wrapping the line in quotes or no quotes, but it doesn't seem to make any difference.

**Comment by lokst** [comment]
The fix is pending the update of the kubernetes-orb: https://github.com/CircleCI-Public/kubernetes-orb/pull/11

**Comment by lokst** [comment]
@parkerjim The issue should now be resolved in version 0.2.3 of the aws-eks orb 🙂
